samples/snippets/csharp/how-to/strings/ParseStringsUsingSplit.cs

```diff
@@ -101,10 +97,10 @@ private static void SplitUsingStrings()
             string[] words = text.Split(separatingChars, System.StringSplitOptions.RemoveEmptyEntries );  
             System.Console.WriteLine("{0} substrings in text:", words.Length);  
     
-            foreach (var word in words)
-            {
-                System.Console.WriteLine(word);
-            }
+            foreach (string s in words)  
+            {  
+                System.Console.WriteLine(s);  
+            }  
```

Comment:
Why does it look as if the changes of PR #4467 would be reverted upon the merge into master? Or I misunderstand something?
The same comment is true for the `docs/csharp/how-to/parse-strings-using-split.md` file changes.

Reply:
I need to rebase before this merges.